### DISCUSSION TITLE
[LLHLS] Possiblity to have relative paths

### DISCUSSION THREAD

**Discussion by p4xx07** [OvenMediaLabs/OvenMediaEngine#1357] [Help]
**Describe the bug**
<img width="873" alt="image" src="https://github.com/AirenSoft/OvenMediaEngine/assets/50495900/30211993-fd22-4c3c-8ac0-57a66876384b">
here the returned path is not relative and I would like an option to have it relative.

from /app/81231ff4-2376-4363-86bc-93b2ff73b0b7/chunklist_0_video_iT0RSmln_llhls.m3u8?session=44_MqRutlvI
to chunklist_0_video_iT0RSmln_llhls.m3u8?session=44_MqRutlvI

is this possibile?

Thanks and have a great day.

**Comment by getroot** [comment]
This is not an official feature (hence undocumented), but try setting the `HLSChunklistPathDepth `option to 0 on your playlist.

```
<Playlist>
	<Name>Paid</Name>
	<FileName>premium</FileName>
	<Options>
		<WebRtcAutoAbr>true</WebRtcAutoAbr> 
		<HLSChunklistPathDepth>0</HLSChunklistPathDepth>
	</Options>
	<Rendition>
		<Name>bypass</Name>
		<Video>bypass_video</Video>
		<Audio>bypass_audio</Audio>
	</Rendition>
</Playlist>
```
